installations.md

```diff
@@ -109,7 +109,7 @@ The flag `-o` is a short for `--open`, which will open your browser in the right
 
 You should see the page like this:
 
-![image](https://github.com/bluebirrrrd/todo-list-tutorial/blob/master/assets/installation-result.png)
```

Comment:
You can also use here `https://github.com/ng-girls/todo-list-tutorial/raw/master/assets/installation-result.png` (it redirects to your link, but it's used as link inside github interface)

Reply:
Ok, thank you. :)